component-samples/aio/src/main/java/org/fidoalliance/fdo/sample/AioConfigLoader.java

```diff
@@ -0,0 +1,56 @@
+package org.fidoalliance.fdo.sample;
+
+import org.apache.commons.configuration2.EnvironmentConfiguration;
+import org.apache.commons.configuration2.FileBasedConfiguration;
+import org.apache.commons.configuration2.PropertiesConfiguration;
+import org.apache.commons.configuration2.SystemConfiguration;
+import org.apache.commons.configuration2.builder.FileBasedConfigurationBuilder;
+import org.apache.commons.configuration2.builder.fluent.Parameters;
+import org.apache.commons.configuration2.ex.ConfigurationException;
+
+public class AioConfigLoader {
+
+  private static final String configurationFile = "application.properties";
```

Comment:
Other components use .env files for loading the configurations. We should change this to follow that pattern.

Reply:
aio uses .env same as other components.  It will use application.properties only if present - all other component have this same definition